templates/download/cloud/tabs/_azure.html

```diff
@@ -3,11 +3,12 @@
      role="tabpanel"
      id="azure-tab"
      aria-labelledby="azure">
-  <div class="col-3 col-medium-3 p-section--shallow">
-    {{ image(url="https://assets.ubuntu.com/v1/75ae0395-azure.png",
+  <div class="col-3 col-medium-3 p-section--shallow"
+       style="margin-top: -13%">
+    {{ image(url="https://assets.ubuntu.com/v1/b009a1df-microsoft-azure-logo-light.png",
```

Comment:
```suggestion
    {{ image (url="https://assets.ubuntu.com/v1/b009a1df-microsoft-azure-logo-light.png",
```

Reply:
It's a bit tricky here considering the logo dimensions and white margins, what specific issues do you think formatting has?

Reply:
As in DJlint formatting